notebooks/mos_spectroscopy_advanced/MOSspec_advanced.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>The introduction of the notebook is scientific and informative by nature, and I found it to be highly engaging and insightful.</p><p>However, including a section in the notebook outlining the intended learning outcomes or goals would help give readers a clear understanding of what they can expect to gain from reading your notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/117/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Included the workflow explicitly and the aim of the notebook.</p>